trying to do something

DmitryTkachov opened the pull request.

DmitryTkachov committed:
trying to do something

artemkhalilyaev commented on ToDoList/ToDoList/ViewController.swift:
```diff
@@ -47,61 +48,75 @@ class ViewController: UIViewController {
+        return 60
+    }
+    func tableView(_ tableView: UITableView, editingStyleForRowAt indexPath: IndexPath) -> UITableViewCell.EditingStyle {
+        return.none
```
add a whitespace

artemkhalilyaev commented on ToDoList/ToDoList/ViewController.swift:
```diff
@@ -47,61 +48,75 @@ class ViewController: UIViewController {
+        return cell
+    }
+//---------------------------------------------------------------------------------------------
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
```
That's wrong and pointless

artemkhalilyaev reviewed (changes requested):
You should create 2 arrays, one array for every section
In first you will store undone tasks and in the second one done tasks

DmitryTkachov committed:
added more interesting things

added more interesting things

DmitryTkachov committed:
added Tab bar controller and other..

1. added Tab bar controller
2. added 2nd NavController and new VC with collection view
3. added realm DB in project
